src/parent-controller.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.parent', ['openfin.window'])
+        .controller('ParentCtrl', ['$scope', 'windowCreationService', function($scope, windowCreationService) {
+            var config = {
+                'autoShow': true,
+                'minWidth': 918,
+                'minHeight': 510,
+                'defaultWidth': 1280,
+                'defaultHeight': 720,
+                'frame': false,
+                'url': 'index.html'
+            };
+
+            windowCreationService.ready(function() {
+                // TODO: Restore correct window(s)
+                windowCreationService.createWindow(config, function(newWindow) {
+                    newWindow.show();
+                });
+
+                $scope.$on('updateFavourites', function(event, data) {
```

Comment:
I've seen `window.storeService` etc used in places, would it not be something like `parentBindings.store.save()`, for example? It'll still be a singleton as the hidden parent will be the only one creating it, and all children would have the same bindings. Injection logic should still be the same, using the `super hack`